src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerProjectsSelector.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import FormikSelect from '../../../../../components/formik-santiment-ui/FormikSelect'
+
+export const TriggerProjectsSelector = ({
+  projects = [],
+  onChange,
+  metaFormSettings,
+  target,
+  setFieldValue
+}) => {
+  const { ethAddress, target: defaultAsset } = metaFormSettings
+
+  const options = projects.map(asset => ({
+    label: asset.slug,
+    value: asset.slug
+  }))
```

Comment:
```
projects.map(({slug})) => ({
    label: slug,
    value: slug
  }))
```
will be better

Reply:
and why you devided to two fields? (just interest)

Reply:
It used in selects, which requires 'label' and 'value' for every option.